curriculum/challenges/english/blocks/lab-planets/68e76b1ce8216249d04df670.md

```diff
@@ -0,0 +1,316 @@
+
+```js
+({
+    test: () => runPython(
+`
+assert _Node(_code).find_class('Planet').find_function('__init__').has_args('self, name, planet_type, star')
+
+`
+    )
+})
+```
+
+The `__init__` function should raise a `ValueError` with the message `name, planet type, and star must be strings` if the first argument passed in is not a string type.
+
+```js
+({
+    test: () => runPython(
+`
+try:
+  planet = Planet(8, "planet", "sun")
+except ValueError as e:
+  assert str(e) == "name, planet type, and star must be strings"
+else:
+  assert False, "Expected to raise error..."
+`
+    )
+})
+```
+
+The `__init__` function should raise a `ValueError` with the message `name, planet type, and star must be strings` if the second argument passed in is not a string type.
+
+```js
+({
+    test: () => runPython(
+`
+try:
+  planet = Planet("Name", [], "sun")
+except ValueError as e:
+  assert str(e) == "name, planet type, and star must be strings"
+else:
+  assert False, "Expected to raise error..."
+`
+    )
+})
+```
+
+The `__init__` function should raise a `ValueError` with the message `name, planet type, and star must be strings` if the third argument passed in is not a string type.
+
+```js
+({
+    test: () => runPython(
+`
+try:
+  planet = Planet("Name", "planet", 0)
+except ValueError as e:
+  assert str(e) == "name, planet type, and star must be strings"
+else:
+  assert False, "Expected to raise error..."
+`
+    )
+})
+```
+
+The `__init__` function should raise a `ValueError` with the message `name, planet_type, and star must be non-empty strings` if the first argument passed in is an empty string.
+
+```js
+({
+    test: () => runPython(
+`
+try:
+  planet = Planet("", "planet", "sun")
+except ValueError as e:
+  assert str(e) == "name, planet_type, and star must be non-empty strings"
+else:
+  assert False, "Expected to raise error..."
+`
+    )
+})
+```
+
+The `__init__` function should raise a `ValueError` with the message `name, planet_type, and star must be non-empty strings` if the second argument passed in is an empty string.
+
+```js
+({
+    test: () => runPython(
+`
+try:
+  planet = Planet("Name", "", "sun")
+except ValueError as e:
+  assert str(e) == "name, planet_type, and star must be non-empty strings"
+else:
+  assert False, "Expected to raise error..."
+`
+    )
+})
+```
+
+The `__init__` function should raise a `ValueError` with the message `name, planet_type, and star must be non-empty strings` if the third argument passed in is an empty string.
+
+```js
+({
+    test: () => runPython(
+`
+try:
+  planet = Planet("Name", "planet", "")
+except ValueError as e:
+  assert str(e) == "name, planet_type, and star must be non-empty strings"
+else:
+  assert False, "Expected to raise error..."
+`
+    )
+})
+```
+
+You should assign the values passed in to the instance attributes `name`, `planet_type`, and `star` in the `__init__` method.
```

Comment:
Considering that this is the first lab where campers deal with classes and objects, this test could be split in three and with hints like "You should assign `name` to `self.name` in the `__init__` method.